services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from django.utils.dateparse import parse_date
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
+                         cinema_hall_id: int) -> MovieSession:
+    movie=Movie.objects.get(id=movie_id)
+    cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+
+    return MovieSession.objects.create(
+        movie_show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
+
+def get_movies_sessions(session_date: str) -> MovieSession:
+    if session_date:
+        date=parse_date(session_date)
+        return MovieSession.objects.filter(show__time__date=date)
+    else:
+        return MovieSession.objects.all()
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+def update_movie_session(session_id: int,
+                         show_time: int | float,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
```

Comment:
The type hint for `show_time` in `update_movie_session` is incorrect. It should be a `datetime`, not `int | float`, as it represents a date and time.